test/asr/check_asr.sh

```diff
@@ -21,6 +21,18 @@ help()
    echo "   h     Print this Help."
 }
 
+# Writes the XS3 TestMode register in the JTAG domain to reboot the device into JTAG-boot mode. 
+# Unsets the 'reboot' bit to release the chip from reset and waits for connection.
+# Will be Fixed in XTC > 15.3.0 (Bugzilla ID 18895).
```

Comment:
Given that this release updates the repository to use tools 15.3.0, is this function still needed?

This comment applies to multiple instances of this function which appear in different .sh files.

Reply:
Sorry, my mistake.  I read the comment as 'Will be fixed in XTC => 15.3.0'.